docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -12,26 +12,32 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Yugabyte Platform can check universes for issues that may affect deployment. Should problems arise, Yugabyte Platform can automatically issue alert notifications.
```

Comment:
Accidental removal?

Reply:
This is weird.  
Preview:
![Screen Shot 2021-12-02 at 8 31 00 AM](https://user-images.githubusercontent.com/77016159/144431703-a41862d0-7af5-44bb-8a95-5670e7383d81.png)

Localhost:
![Screen Shot 2021-12-02 at 8 31 13 AM](https://user-images.githubusercontent.com/77016159/144431768-9af0d972-141a-432e-90a3-f276ac79ce50.png)